k8s/sandbox/common/idam-sprod/idam-web-admin.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-web-admin
+  namespace: idam-sprod
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:preview-*
+spec:
+  releaseName: idam-web-admin
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: idam-web-admin
+    version: 0.2.3
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/web-admin:preview
+      ingressHost: idam-web-admin-sbox.service.core-compute-sprod.internal
+      aadIdentityName: idam-sprod
```

Comment:
You'll need to add environment overrides like https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/idam/idam-web-admin.yaml#L24-L26